### PR TITLE
node name resolver skip on identifier

### PR DESCRIPTION
**Pull request by mssimi**
Closes: #4239

### CODE REVIEW

File: rules/php70/tests/Rector/FuncCall/NonVariableToVariableOnFunctionCallRector/Fixture/keep_curly_braces.php.inc [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Rector\Php70\Tests\Rector\FuncCall\NonVariableToVariableOnFunctionCallRector\Fixture;
+
+final class ModelSmokeTest
+{
+    /** @var array|null */
+    private $models;
+
+    public function testGetters(): void
```
Comment (TomasVotruba) [inline rules/php70/tests/Rector/FuncCall/NonVariableToVariableOnFunctionCallRector/Fixture/keep_curly_braces.php.inc:10]: This looked as test at first sight. To me a while to found out it's a fixture :D

Better use dummy name `run()` as in other tests

File: rules/php70/tests/Rector/FuncCall/NonVariableToVariableOnFunctionCallRector/Fixture/keep_curly_braces.php.inc [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Rector\Php70\Tests\Rector\FuncCall\NonVariableToVariableOnFunctionCallRector\Fixture;
+
+final class ModelSmokeTest
```
Comment (TomasVotruba) [inline rules/php70/tests/Rector/FuncCall/NonVariableToVariableOnFunctionCallRector/Fixture/keep_curly_braces.php.inc:5]: The class should be named with same words as in file.

Hm... I know it's not much obvious.
 Could you make a CI rule for that?

Comment (mssimi) [reply rules/php70/tests/Rector/FuncCall/NonVariableToVariableOnFunctionCallRector/Fixture/keep_curly_braces.php.inc]: I will make CI rule in new task.

**Comment by TomasVotruba** [comment]
Thanks :+1:

**Pull request merged**